app/components/crate-sidebar/install-instructions.hbs

```diff
@@ -0,0 +1,76 @@
+{{#if @binNames}}
+  {{#if (is-clipboard-supported)}}
+    <CopyButton
+      @copyText={{this.cargoInstallCommand}}
+      title="Copy command to clipboard"
+      local-class="copy-button"
+    >
+      <span local-class="selectable">{{this.cargoInstallCommand}}</span>
+      {{svg-jar "copy" aria-hidden="true" local-class="copy-icon"}}
+    </CopyButton>
+  {{else}}
+    <code local-class="copy-fallback">
+      {{this.cargoInstallCommand}}
+    </code>
+  {{/if}}
+
+  <p local-class="copy-help">
+    Running the above command will globally install the following binaries:
+    {{#if (eq @binNames.length 1)}}
+      <span local-class="bin-name">{{get @binNames 0}}</span>
+    {{else if (eq @binNames.length 2)}}
+      <span local-class="bin-name">{{get @binNames 0}}</span> and <span local-class="bin-name">{{get @binNames 1}}</span>
```

Comment:
```suggestion
      <span local-class="bin-name">{{get @binNames 0}}</span>, and <span local-class="bin-name">{{get @binNames 1}}</span>
```

_insert link to Vampire Weekend's Oxford Comma song here_

Reply:
I guess in that case we might as well remove the `length == 2` special case 😂

Reply:
nevermind, https://en.wikipedia.org/wiki/Serial_comma says "in a series of three or more terms" 🙈